### PR TITLE
#550 rust_toolchain_repository: add urls to enable project specific rust file server mirrors

### PR DESCRIPTION
**Pull request by dirkboye**
environment variable `STATIC_RUST_URL` will work as before.

### CODE REVIEW

**Comment by dirkboye** [comment]
#550

File: rust/repositories.bzl
Comment (UebelAndre) [inline rust/repositories.bzl:420]: I'm not sure it's the best idea to change the environment variable name. Whats the justification here?

File: rust/repositories.bzl
Comment (UebelAndre) [inline rust/repositories.bzl:673]: Nit, I think users will see the default in the rendered docs

**Review by UebelAndre — CHANGES_REQUESTED** [review]
Two comments but otherwise looking good to me (as a non maintainer 😅)

File: rust/repositories.bzl
Comment (dirkboye) [reply rust/repositories.bzl:420]: i think only the docs were wrong as some lines below the variable "STATIC_RUST_URL" is imported from environment.

File: rust/repositories.bzl
Comment (dirkboye) [reply rust/repositories.bzl:673]: should i remove the default part from the doc?

File: rust/repositories.bzl
Comment (UebelAndre) [reply rust/repositories.bzl:420]: Ah, good catch!

File: rust/repositories.bzl
Comment (UebelAndre) [reply rust/repositories.bzl:673]: I would, I know there are a bunch of similar docs for macro parameters, but I think it'd be better to keep them out of attribute `doc`s.

Comment (dirkboye) [reply rust/repositories.bzl]: done in fixup!

**Review by UebelAndre — APPROVED** [review]

**Comment by UebelAndre** [comment]
Oh, one last thing, can you run `./docs/update_docs.sh` and push the results?

**Comment by dirkboye** [comment]
pushed the re-generated docs!

thanks for the fast review!

**Comment by UebelAndre** [comment]
> pushed the re-generated docs!
> 
> thanks for the fast review!

No problem 😄 Again, I'm not a maintainer so I can't merge it but I like the changes and think they'd be a great addition. Hopefully another maintainer can take a quick look today. @damienmg @illicitonion @dfreese ?

**Review by damienmg — APPROVED** [review]

**Pull request merged**
